src/app/services/services.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { AircraftStatus } from 'src/app/interfaces/aircraft-status';
+import { Trip } from 'src/app/interfaces/trip';
+import { User } from 'src/app/interfaces/user';
+import { Passenger } from 'src/app/interfaces/passenger';
+import { PaymentDetails } from 'src/app/interfaces/payment-details';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class AppService {
+    private departureAircraftStatusUrl = "/assets/mock-data/mockDepartureAircraftStatus.json";
+    private returnAircraftStatusUrl = "/assets/mock-data/mockReturnAircraftStatus.json";
+    private currentTripUrl = "assets/mock-data/mockCurrentTrip.json";
+    private returnTripUrl = "assets/mock-data/mockReturnTrip.json";
+    private domesticTripsUrl = "/assets/mock-data/mockPastDomesticTrips.json";
+    private internationalTripsUrl = "/assets/mock-data/mockPastInternationalTrips.json";
+    private userUrl = "/assets/mock-data/mockUser.json";
+    private passengersUrl = "/assets/mock-data/mockPassengers.json";
+    private paymentDetailsURL = "/assets/mock-data/mockPaymentDetails.json";
+
+    constructor(private http: HttpClient) { }
+
+    private getJSON<T = unknown>(url: string): Observable<T> {
+        return this.http.get<T>(url);
+    }
```

Comment:
I think it is not worth having this function at all, call `this.http.get` directly when needed